src/persist-client/src/cache.rs

```diff
@@ -382,6 +556,9 @@ impl StateCache {
                     if let Some(x) = did_init {
                         // We actually did the init work, don't bother casting back
                         // the type erased and weak version.
+                        if let Some(pubsub_sender) = self.pubsub_sender.as_ref() {
+                            pubsub_sender.subscribe(&shard_id);
```

Comment:
ah hmm seems ideally that the `PubSubSubscriptionToken` would be something that comes back from a `subscribe` call no?